lms/djangoapps/ccx/api/v0/tests/test_views.py

```diff
@@ -153,42 +159,86 @@ def test_authorization(self):
         ]
         # all the auths in the list fail to authorize
         for auth in auth_list:
-            resp = self.client.get(url, {}, HTTP_AUTHORIZATION=auth)
+            resp = self.client.get(self.list_url_master_course, {}, HTTP_AUTHORIZATION=auth)
             self.assertEqual(resp.status_code, status.HTTP_401_UNAUTHORIZED)
-        resp = self.client.get(url, {}, HTTP_AUTHORIZATION=self.auth)
+        resp = self.client.get(self.list_url_master_course, {}, HTTP_AUTHORIZATION=self.auth)
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+    def test_authorization_no_oauth(self):
+        """
+        Check authorization for users logged in without oauth
+        """
+        # create a staff user
+        staff_user = User.objects.create_user('test_staff_user', 'test_staff_user@openedx.org', 'test')
+        # add staff role to the app user
+        CourseStaffRole(self.master_course_key).add_users(staff_user)
+        # create an instructor user
+        instructor_user = User.objects.create_user('test_instructor_user', 'test_instructor_user@openedx.org', 'test')
+        # add staff role to the app user
+        CourseInstructorRole(self.master_course_key).add_users(instructor_user)
+        # create an coach user
+        coach_user = User.objects.create_user('test_coach_user', 'test_coach_user@openedx.org', 'test')
+        # add staff role to the app user
```

Comment:
Is this comment accurate? Shouldn't it be a coach role (maybe I'm getting the terms confused)